plugins/core/src/main/resources/schemas/FedizConfig.xsd

```diff
@@ -205,6 +205,21 @@
             <xs:element ref="reply" />
         </xs:sequence>
     </xs:complexType>
+    
+     <xs:complexType name="SignRequestType">
+        <xs:simpleContent>
+            <xs:extension base="xs:boolean">
+                <xs:attribute name="algorithm" type="signAlgorithmType" />
+            </xs:extension>
+        </xs:simpleContent>
+    </xs:complexType>
+    
+    <xs:simpleType name="signAlgorithmType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="SHA1" />
+            <xs:enumeration value="SHA256" />
+        </xs:restriction>
+    </xs:simpleType>
```

Comment:
The algorithms here are actually digest algorithms. I think it would be more accurate to rename "signAlgorithmType" as "signatureDigestAlgorithmType", and change "algorithm" to "signatureDigestAlgorithm".